prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -123,4 +146,124 @@ class LivdTableDownload() : CliktCommand(
             }
         return allLinks.filter { it.contains(partialHref) }
     }
+
+    /**
+     * Extracts LIVD table from sheet [sheetName] of the input Excel format [inputfile] file, converts to
+     * csv format, and output to the CSV format [outputfile] file.
+     * @param sheetName is the LOINC Mapping sheet from the downloaded LOINC data code file.
+     * @param inputfile is the input Excel file name.
+     * @param inputfile is the output CSV file name.
+     * @return true for success and false for failure.
+     */
+    private fun extractLivdTable(sheetName: String, inputfile: String, outputfile: String): Boolean {
+        //
```

Comment:
No need to add the extra lines with //.  A single line is enough and is consistent with other errors.

Reply:
Done.  Thanks